### ISSUE TITLE
Fix empty label values in facetResults JSON output

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/biocache-service#219]
This appears to be happening in a number of web-service endpoints where there is a `facetResults` object returned. E.g. 

```
"facetResults": [
    {
      "fieldName": "genus",
      "fieldResult": [
        {
          "label": "Pogona",
          "i18nCode": "genus.Pogona",
          "count": 17,
          "fq": "genus:\"Pogona\""
        },
        {
          "label": "",
          "i18nCode": "genus.novalue",
          "count": 284,
          "fq": "-genus:*"
        }
      ]
    }
  ]
```

Note the `"label": ""` line.

Fix is to include a sensible value in here which should read the value from the provided i18n code of `genus.novalue` (in this case), for the default locale).  

TL;DR check there are entries for all the fields with a `.novalue` entry in i18n file. Code may need to provide a generic term for when there is no entry in the i18n properties.

**Comment by djtfmartin** [comment]
fixed for 2.x. Closing
